js/plugins/vertexai/src/imagen.ts

```diff
@@ -222,8 +258,15 @@ export function imagenModel(
       const instance: ImagenInstance = {
         prompt: extractText(request),
       };
-      if (extractPromptImage(request))
-        instance.image = { bytesBase64Encoded: extractPromptImage(request)! };
+      const baseImage = extractBaseImage(request);
+      if (baseImage) {
+        instance.image = { bytesBase64Encoded: baseImage };
+      }
+      if (extractMaskImage(request)) {
+        instance.mask = {
+          image: { bytesBase64Encoded: extractBaseImage(request)! },
```

Comment:
Should this be a call to `extractMaskImage(...)`?

Reply:
lol, good catch. that's what I get for last minute copy-paste "fixes"